### DISCUSSION TITLE
Creatin FFD point in ICEM CFD

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#260] [General]
Hi,

Could you please explain to me how to extract FFD from ICEM? I draw the points and curves in the geometry section of ICEM, but I can not export it or save it in xyz format. I would appreciate it if you help me in this regard.

**Comment by friedenhe** [comment]
This should be a pretty standard process for ICEM-CFD. Google: "icem cfd export mesh to plot3d format"
